<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorsConfigForm</class>
 <widget class="QWidget" name="EditorsConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customize Web Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_activeXmlGroupBox">
     <property name="title">
      <string>Active completion on XML</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="m_addClosedBaliseCheckBox">
        <property name="text">
         <string>&amp;Add closed balise automatically</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="m_addDefaultAttributeCheckBox">
        <property name="text">
         <string>Add default a&amp;ttribute automatically</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="m_addDefaultSubBaliseCheckBox">
        <property name="text">
         <string>Add default sub-&amp;balise automatically</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_javaScriptGroupBox">
     <property name="title">
      <string>Active completion on JavaScript</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Auto close bracket</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
